templates/commands/render_test.go

```diff
@@ -76,7 +76,7 @@ func TestParseFlags(t *testing.T) {
 				flagSpec:           "./spec.yaml",
 				flagDest:           ".",
 				flagGitProtocol:    "https",
-				flagInputs:         nil,
+				flagInputs:         map[string]string{},
```

Comment:
This was actually a bug in the CLI. Chen fixed it in https://github.com/abcxyz/pkg/pull/128, and the update brought it in.

Reply:
👍